flexmeasures/data/scripts/clean_database.sh

```diff
@@ -13,6 +13,14 @@ function is_database() {
 function create_database() {
  echo "Creating a new database ..."
  sudo -i -u postgres createdb -U postgres $1
+ if [[ -n "$2" ]]; then
+   echo "Creating database user ..."
+   sudo -i -u postgres createuser --pwprompt -U postgres $2
```

Comment:
I'm not sure how to do it, but when the passwords don't match, the user won't be created and we should stop the script. 
I guess we should check its return type.

Reply:
dont.